We currently don't ship glx.pc in any Solaris package.
We can remove this once we upgrade to a version of mesa or glvnd that
delivers it.

--- mesa-demos-9.0.0/meson.build
+++ mesa-demos-9.0.0/meson.build
@@ -99,7 +99,7 @@ required : dep_x11.found())
 endif
 
 dep_glx = dependency('glx', required: false, disabler : true)
-if not dep_glx.found() and host_machine.system() == 'darwin'
+if not dep_glx.found() and host_machine.system() in ['darwin', 'sunos']
   # xquartz doesn't have a glx.pc, but it does have the header. And all the
   # symbols reside in libGL, so let's just use that.
   if cc.check_header('GL/glx.h', dependencies: dep_x11)
